irishspell.py

```diff
@@ -94,10 +95,12 @@ def read_file(filename):
         
         if isValidWord:
             irishWord = testWord.strip()
-
-          #print(irishWord)
-            f.write(f"{correction(irishWord)}\n")
-            # print(correction(irishWord))
+            correct_word = correction(irishWord)

```

Comment:
Nice one Sai!! This is terrific. The case issue for every first letter of the word is now handle with this code! Well done!